test/uk/gov/hmrc/merchandiseinbaggagefrontend/pagespecs/CheckYourAnswersPageSpec.scala

```diff
@@ -18,13 +18,18 @@ package uk.gov.hmrc.merchandiseinbaggagefrontend.pagespecs
 
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.merchandiseinbaggagefrontend.model.core.{DeclarationJourney, TaxCalculations}
+import uk.gov.hmrc.merchandiseinbaggagefrontend.pagespecs.pages.{CheckYourAnswersPage, InvalidRequestPage}
 import uk.gov.hmrc.merchandiseinbaggagefrontend.service.CalculationService
 import uk.gov.hmrc.merchandiseinbaggagefrontend.stubs.CurrencyConversionStub.givenCurrencyIsFound
 
 import scala.concurrent.Future
 
-class CheckYourAnswersPageSpec extends BasePageSpec {
```

Comment:
This spec is probably likely to remain a snowflake as this page is unique and complex.